service/raft/raft_group0_client.cc

```diff
@@ -13,6 +13,9 @@
 
 #include "frozen_schema.hh"
 #include "schema_mutations.hh"
+#include "raft/broadcast_tables/experimental/lang.hh"
+#include "idl/experimental/broadcast_tables_lang.dist.hh"
+#include "idl/experimental/broadcast_tables_lang.dist.impl.hh"
```

Comment:
I'm unhappy about mixing in all this stuff into group0, but okay as it's experimental (and destined to be replaced, not graduated into production). I retract my comment about moving it ourside raft/, it's too deeply entangled.

Reply:
We should move outside raft/, because raft/ contains the library and only the library; the users of the library should be somewhere else. All our users reside in service/raft/ so I think that's the proper place

Reply:
I retract my retraction.

Reply:
Moved from `raft/broadcast_tables` to `service/broadcast_tables/`.